api/documentation/src/pages/change-logs.md

```diff
@@ -20,6 +20,12 @@ title: Pass Culture API change logs
 ## October 2024
 
 ### Addresses endpoints
+
+:::info
+New endpoints allow you to specify that your offer is available at a location different from your venue. Learn how to use them in [**our documentation**](/rest-api#tag/Addresses).
```

Comment:
(détail) 
New endpoints let you specify that your offer is available at a location different from your venue. Learn how to use them in our documentation.

Reply:
Alors j'avais demandé à chat GPT (globalement je lui demande toujours de reformuler) et je crois qu'il m'avait répondu que `allow` c'est plus formel que `let`)